SortAndRemoveOnSave/TextEditorExtension.cs

```diff
@@ -20,8 +25,21 @@ protected override void Initialize()
 			DocumentContext.Saved += Document_Saved;
 		}
 
+
+
+
 		private void Document_Saved(object sender, EventArgs e)
 		{
+			var x = IdeApp.CommandService.GetCommands().ToList();
```

Comment:
Do we need that? Looks like these lines were only for test?

Reply:
Only for test, i removed it.